docs/java-api/index.asciidoc

```diff
@@ -44,12 +44,12 @@ You need to also include Log4j 2 dependencies:
 <dependency>
     <groupId>org.apache.logging.log4j</groupId>
     <artifactId>log4j-api</artifactId>
-    <version>2.7</version>
+    <version>2.8.2</version>
```

Comment:
It should be possible to define an attribute / variable for the log4j version and change it only at one place